src/components/organisms/SfCarousel/SfCarousel.js

```diff
@@ -0,0 +1,39 @@
+import Vue from "vue";
+import SfCarouselItem from "@/components/organisms/SfCarousel/_internal/SfCarouselItem.vue";
+import Glide from "@glidejs/glide";
+
+Vue.component("SfCarouselItem", SfCarouselItem);
+
+export default {
+  name: "SfCarousel",
+  data() {
+    return {
+      glide: null
+    };
+  },
+  props: {
+    perPage: {
+      type: Number,
+      default: 4
+    }
+  },
+  methods: {},
+  mounted: function() {
+    const glide = new Glide(this.$refs.glide, {
+      type: "carousel",
+      perView: this.perPage,
+      breakpoints: {
+        480: {
+          perView: 1
+        },
+        600: {
+          perView: 2
+        }
+      }
+    });
+
+    glide.mount();
+
```

Comment:
@przemyslawspaczek I think you should handle pagination manually here, glidejs supports moving the items one by one only, and imo the best experience is moving all viewed items, agree?
I'm not sure if I was clear, take a look [here as example](https://github.com/ecomclub/storefront/blob/master/template/js/src/utils.js#L88)...

Reply:
```
glide.on("run.before", move => {
      if (!this.movePerPage) return;

      const { perView } = glide.settings;
      const { direction } = move;

      move.steps = direction === "<" ? perView : -perView;
    });
```
I added option movePerPage, by default set to single element

Reply:
Great, but imo pagination would be set by default as it's the most common use case, I think it works with pagination in all the ecommerces I know...

```js
move.steps = direction === "<" ? perView : -perView;
```

Did it work well on your tests?
If I remember correctly, when I implemented this same feature with glide, I tried this way but I had some bugs (on Firefox) depending on the direction of the movement and/or end of carousel (loop back).
So I chose to hard set the index (this solved in my case):

```js
move.direction = '='
move.steps = newIndex
```

Reply:
Yes its's work well, on Firefox too. You have right, Ok, I set to default scrollPerPage. You have right